src/dipdup/project.py

```diff
@@ -373,3 +374,6 @@ def _render(answers: Answers, template_path: Path, output_path: Path, force: boo
         header=CODEGEN_HEADER,
     )
     write(output_path, content, overwrite=force)
+
+
+reload_env()
```

Comment:
This will make variables loaded twice on import. Instead, call this function in `dipdup.cli._load_env_files`